assets/pages/new-contributions-landing/components/ContributionAmount.jsx

```diff
@@ -7,64 +7,74 @@ import { connect } from 'react-redux';
 
 import { amounts, type Amount, type Contrib } from 'helpers/contributions';
 import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
-import { type CountryMetaData } from 'helpers/internationalisation/contributions';
+import { type IsoCurrency, type Currency, type SpokenCurrency, currencies, spokenCurrencies, detect } from 'helpers/internationalisation/currency';
+import { classNameWithModifiers } from 'helpers/utilities';
 
 import SvgDollar from 'components/svgs/dollar';
 
+import { type State } from '../contributionsLandingReducer';
 
 // ----- Types ----- //
 
 type PropTypes = {
-  countryGroupDetails: CountryMetaData,
```

Comment:
You're still passing this through as a prop aren't you?

Reply:
Good catch!